src/components/RadioButtonGroup/index.js

```diff
@@ -7,7 +7,7 @@ export default class RadioButtonGroup extends Component {
   constructor(props) {
     super(props);
     this.listOfItems = props.listOfItems;
-    this.state = {optionsVal: ''};
+    this.state = {optionsVal: props.checked};
```

Comment:
In case there is no props of checked, can we set the value to empty string? Lets not make it always mandatory? @ldgarber